### PR TITLE
Feat/pages

### PR DESCRIPTION
**Pull request by valieff**
1. Footer navigation now shows links to all text pages
2. HTML content for text pages is fetched from CMS

### CODE REVIEW

File: lib/get-all-textpages.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const getAllTextPagesQuery = gql`
+  query AllTextPagesQuery() {
+    pages() {
+      slug
+      title
+    }
+  }
+`
+
+export const getAllTextPages = async (): Promise<any> => {
```
Comment (sorokinvj) [inline lib/get-all-textpages.ts]: lets remove `any` and create a proper interface that will describe the response `getAllTextPages` returns

File: types.ts [hunk 2]
```diff
@@ -59,6 +59,21 @@ export interface Page {
   name: string
   type: string
   slug: string
+  title: string
+}
+
+export interface TextPage {
+  // TODO: rename to Something
```
Comment (sorokinvj) [inline types.ts]: lets remove this comment

**Review by sorokinvj — CHANGES_REQUESTED** [review]
Vova, awesome work 🎸 
Just a couple of comments in the code, please lets also align the page with the logo - it seems a bit wider than needed now

File: lib/get-all-textpages.ts (hunk 1 above)
Comment (valieff) [reply lib/get-all-textpages.ts]: What is the correct way? Other pages (products, collections) use a single generic type only.

1. Add a special interface that only contains keys/values used in paths

`interface PagePath {
  slug: string
  title: string
}`

2. Use a more generic type:

`interface TextPage {
  id: string
  name: string
  type: string
  slug: string
  title: string
  page: {
    content: {
      html: string
    }
  }
}`

Comment (sorokinvj) [reply lib/get-all-textpages.ts]: interface TextPage seems what we need!

**Review by sorokinvj — APPROVED** [review]
Awesome!

**Pull request merged**